### PR TITLE
Test with CPU in GH actions

### LINKED ISSUE #510: Introduce CPU-only tests in GitHub Actions
Now that markers are added in #509, we can selectively run CPU tests.

### PR DESCRIPTION
**Pull request by emcastillo**
Closes #510 

Discuss: Python versions

### CODE REVIEW

File: .github/workflows/test-cpu.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        include:
+           - torch: '1.8.*'
+             torchvision: '0.9.*'
```
Comment (emcastillo) [inline .github/workflows/test-cpu.yml:12]: Should we set python versions in the matrix, I think yes :)

**Comment by kmaehashi** [comment]
I rather think only one variant (fixed PyTorch & Python version) is enough because (1) CPU-only test is to confirm that test markers are correctly specified to each test (2) We will run FlexCI for all PRs which runs every test (3) Slots of GitHub Actions is shared within the OSS products under the `pfnet` organization so I don't want to monopolize it 😅

**Comment by emcastillo** [comment]
only one test now :)

**Review by kmaehashi — APPROVED** [review]
LGTM!

**Pull request merged**